lib/serum/dev_server/prompt.ex

```diff
@@ -105,6 +109,13 @@ defmodule Serum.DevServer.Prompt do
     :detach
   end
 
+  defp do_run_command("open", _options) do
+    case @command_handler.open_url("http://localhost:#{@service.port()}") do
+      :ok -> :ok
+      _ -> put_err(:warn, "Can't open browser")
```

Comment:
This line is not covered by our tests. My suggestion is to add another test case in `prompt_test.exs`, where `Serum.DevServer.CommandHandler.Mock.open_url/1` returns `:error`.

```elixir
test "prints a warning when failed to open a browser" do
  Serum.DevServer.CommandHandler.Mock
  |> expect(:open_url, fn -> "http://localhost:8080" -> :error end)

  # ...
end
```